src/main/java/com/google/api/generator/gapic/model/GapicClass.java

```diff
@@ -62,6 +72,8 @@ abstract static class Builder {
 
     abstract Builder setSamples(List<Sample> samples);
 
+    abstract Builder setDefaultHost(String defaultHost);
```

Comment:
Looking at the PR again, I noticed that it's not the default host we are setting here, we are actually the apiShortName that are parsed from default host, do you mind renaming this field to `apiShortName`?

Reply:
You are right, I was confused with what's parsed and what's being set in GapicClass. SpringCodeGen may want to reuse the parsing logic so I was thinking we might want to move the parsing logic earlier in the process, but that can be done in a different PR as well.